Dangerfile.swift

```diff
@@ -1,39 +1,95 @@
+//    !pullRequestBody.contains("🤔 Why") ||
+//    !pullRequestBody.contains("🛠 How") {
+//    warn("""
+//    Pull request description is missing required information:
+//     - 📲 What
+//     - 🤔 Why
+//     - 🛠 How
+//     - 👀 See
+//    Please use the pull request:template:
+//    https://github.com/kickstarter/ios-oss/blob/master/.github/PULL_REQUEST_TEMPLATE.md
+//    """)
+//}
 
-// import Danger
-// let danger = Danger()
+// MARK: - 2 - Large Pull Request
 
-// var bigPRThreshold = 600;
-// if (danger.github.pullRequest.additions + danger.github.pullRequest.deletions > bigPRThreshold) {
-//   warn('> Pull Request size seems relatively large. If this Pull Request contains multiple changes, please split each into separate PR will helps faster, easier review.');
-// }
+var largePRLineCount = 500;
+let additions: Int = danger.github.pullRequest.additions ?? 0
+let deletions: Int = danger.github.pullRequest.deletions ?? 0
 
+if ((additions + deletions) > largePRLineCount) {
+  warn("""
+    Number of pull request changes is larger than 500! Consider breaking out future changes into smaller pull requests.
+    """)
+}
 
-// // MARK: - 10 - Dispatch Async
+// MARK: - 3 - SwiftLint
 
-// func checkDispatchSyncIsNotCalledOnMain() {
-//   let excludedFiles = [
-//     "Dangerfile"
-//   ]
-//   for file in filter(files: filesChanged, with: [.swift]) {
-//     guard excludedFiles.allSatisfy({ !file.contains($0) }) else { continue }
-//     let fileLines = read(file: file, danger: danger)
+SwiftLint.lint(inline: true, strict: true, lintAllFiles: true)
 
-//     for (index, line) in fileLines.enumerated() {
-//       if line.contains("DispatchQueue.main.sync") {
-//         let link = "https://stackoverflow.com/questions/44324595/difference-between-dispatchqueue-main-async-and-dispatchqueue-main-sync"
-//         warn(message: "Please async on the main queue. [More information](\(link))",
-//              file: file,
-//              line: index + 1)
-//       }
-//     }
-//   }
-// }
+// MARK: - 4 - Celebrate Milestones
+
+let pullRequestMilestones = [3, 10, 100]
+let currentPRNumber = danger.github.pullRequest.number
+if pullRequestMilestones.contains(currentPRNumber) {
+    let githubHandle = danger.github.pullRequest.user.login
+    message("Congratulations \(githubHandle)! You've made the \(currentPRNumber)th Pull Request!")
+}
+    
+ // MARK: - 10 - Dispatch Async
+
+ func checkDispatchSyncIsNotCalledOnMain() {
+   let excludedFiles = [
+     "Dangerfile"
+   ]
+   for file in filter(files: danger.git.modifiedFiles + danger.git.createdFiles, with: [.swift]) {
+     guard excludedFiles.allSatisfy({ !file.contains($0) }) else { continue }
+     let fileLines = read(file: file, danger: danger)
+
+     for (index, line) in fileLines.enumerated() {
+       if line.contains("DispatchQueue.main.sync") {
+         let link = "https://stackoverflow.com/questions/44324595/difference-between-dispatchqueue-main-async-and-dispatchqueue-main-sync"
```

Comment:
<!--
  1 failure:  Line should be 12...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Dangerfile.swift;
  Line: 62;
-->
- :no_entry_sign: Line should be 120 characters or less: currently 138 characters (`line_length`)